fml/message_loop_impl.cc

```diff
@@ -11,35 +13,29 @@
 #include "lib/fxl/build_config.h"
 
 #if OS_MACOSX
-
 #include "flutter/fml/platform/darwin/message_loop_darwin.h"
-using PlatformMessageLoopImpl = fml::MessageLoopDarwin;
-
 #elif OS_ANDROID
-
 #include "flutter/fml/platform/android/message_loop_android.h"
-using PlatformMessageLoopImpl = fml::MessageLoopAndroid;
-
 #elif OS_LINUX
-
 #include "flutter/fml/platform/linux/message_loop_linux.h"
-using PlatformMessageLoopImpl = fml::MessageLoopLinux;
-
 #elif OS_WIN
-
 #include "flutter/fml/platform/win/message_loop_win.h"
-using PlatformMessageLoopImpl = fml::MessageLoopWin;
-
-#else
-
-#error This platform does not have a message loop implementation.
-
 #endif
 
 namespace fml {
 
 fxl::RefPtr<MessageLoopImpl> MessageLoopImpl::Create() {
-  return fxl::MakeRefCounted<::PlatformMessageLoopImpl>();
+#if OS_MACOSX
```

Comment:
Not in this patch, but I wonder if we should rename this to OS_DARWIN when we finally settle on one set of OS macros. Every time I see this #define I end up having to look up whether it's macOS and iOS or just macOS.